docs/manual/developer_guide.adoc

```diff
@@ -594,9 +594,18 @@ For example, to check which rules in RHEL8 OSPP profile are missing remediations
 
 ----
 $ ./build_product rhel8
-$ ./build-scripts/profile_stats.py --missing-fixes --profile ospp --benchmark build/ssg-rhel8-xccdf.xml
+$ ./build-scripts/profile_stats.py stats --missing-fixes --profile ospp --benchmark build/ssg-rhel8-xccdf.xml
 ----
 
+The tool also can create diff between YAML profiles and generate a new one.
+
+For example, to generate a profile diff between two profiles, run this command:
+----
+$ ./build-scripts/profile_stats.py diff --profile1 rhel7/profiles/ospp.profile --profile2 rhel7/profiles/pci-dss.profile 
+----
+
+A new profile will be generated in the current directory with the name, in this case, ospp-compared_to-pci-dss.profile
```

Comment:
I would rephrase this, because I would expect an output similar to `diff` output, but in fact it generates a new profile containing only rules from profile1 that are not contained in profile2.

Reply:
I changed the approach of the feature to subtraction instead of diff.